tsconfig.json

```diff
@@ -9,14 +9,27 @@
 		"sourceMap": true,
 		"rootDirs": [
 			"src",
-			"test"
+			"test",
+			"styles"
```

Comment:
Are you sure we need this here? There's no TS files in `styles`.

Reply:
i dont remmber

Reply:
I think we don't, can you try removing it and build?